cypress/tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
+  "extends": "../tsconfig",
   "compilerOptions": {
-    "target": "es5",
-    "lib": ["es5", "dom"],
     "types": ["cypress", "@testing-library/cypress"]
   },
-  "include": ["**/*.ts"]
+  "include": ["**/*.ts"],
+  "exclude": [],
```

Comment:
Is this exclude necessary?

Reply:
Absolutely, since we are extending the root `tsconfig.json` and that one excludes `cypress` we need to overwrite the exclude prop for the cypress to run.